test/old_tests/UnitTests/async.cpp

```diff
@@ -22,7 +22,7 @@ namespace
 
     IAsyncAction NoSuspend_IAsyncAction()
     {
-        co_await resume_after(0s);
+        co_await 0s;
```

Comment:
What does this change have to do with ripping out the low-level hooks?

Reply:
This workaround was added because of the low level hooks preventing Clang from building this code.

So removing resume_after helps make sure that we have coverage for this functionality on Clang.